src/parser/metrics/RealLinesOfCode.ts

```diff
@@ -56,14 +56,16 @@ export class RealLinesOfCode implements Metric {
     }
 
     async calculate(parseFile: ParseFile, tree: Parser.Tree): Promise<MetricResult> {
+        let rloc = 0;
         const cursor = tree.walk();
-        // Assume the root node is always some kind of program/file/compilation_unit stuff
-        cursor.gotoFirstChild();
-        const sureCodeLines = this.walkTree(cursor);
 
-        dlog("Included lines for rloc: ", sureCodeLines);
-
-        const rloc = sureCodeLines.size;
+        // Assume the root node is always some kind of program/file/compilation_unit stuff.
+        // So if there are no child nodes, the file is empty.
+        if (cursor.gotoFirstChild()) {
+            const sureCodeLines = this.walkTree(cursor);
```

Comment:
```suggestion
            const sourceCodeLines = this.walkTree(cursor);
```

Reply:
That was actually no typo. "sure code lines", because we can be sure there is code in them :laughing: 
But yeah, no one will understand that, I will change it.